this_is_the_way.py

```diff
@@ -1,16 +1,15 @@
 import os
 
 
-def this_is_the_way():
-    res_lst = []
-    path = input("Enter path to directory: ")
+def this_is_the_way(directory_path):
+    files_that_start_with_deep = []
```

Comment:
though you shouldn't use abbreviations, you should also try to avoid names that are too long, because you want you code to be easy to read. So try to find the shortest name possible that conveys the purpose of the parameter.
For example: files_start_deep.